mainapp/templates/mainapp/disclaimer.html

```diff
@@ -4,6 +4,6 @@
 {% block content %}
 <div class="row text-center static-page-box">
     <h1 class="text-center">Disclaimer</h1>
-    <p style="font-size:1.7rem;">This portal has been developed for effective collaboration between various stakeholders such as citizen, Government authorities, various NGOs and volunteers. IEEE makes no representation regarding the completeness, accuracy of any information and data contained on this portal or that such information and data will be error-free. Stakeholders are trying to make their best effort to support, any information received through the portal the requests received through this portal doesn’t warrant any commitment on the services. By using this site you acknowledge that you have read and agree to this</p>
+    <p style="font-size:1.7rem;">This portal has been developed to serve as one of the means for effective collaboration between various stakeholders such as citizen, Government authorities, various NGOs and volunteers. IEEE makes <b>NO</b> representation regarding the completeness, accuracy and correctness of any information and data contained on this portal or that such information and data will be completely error-free. Stakeholders are trying to pump in their best of efforts to support, any information received through the portal, the requests received through this portal doesn’t warrant any commitment on the services. By using this site, you acknowledge that you have read and agree to the mentioned document.</p>
```

Comment:
How about this alternative

This portal is meant to help ease collaboration between citizens, government authorities, various NGOs, and volunteers. Like any user-generated collaborative website, IEEE cannot verify the completeness, accuracy, and correctness of the information and data presented on this website. By using this site, you understand and acknowledge that the information presented here is best-effort and cannot be interpreted as a commitment to any of the services requested.